build.gradle

```diff
@@ -57,7 +57,7 @@ subprojects { project ->
   dependencies {
     spinnaker.group('test')
     testCompile spinnaker.dependency('groovy')
-    testCompile 'org.springframework.boot:spring-boot-starter-test'
+    testCompile("org.springframework.boot:spring-boot-starter-test:${spinnaker.version('springBoot')}")
```

Comment:
I think this could also be `testCompile spinnaker.dependency('bootTest')`

Reply:
That looks much better! I'll make the change